ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -24,9 +24,7 @@ StatusDialog {
         id: d
         property string importErrorMessage
         readonly property bool communityFound: (d.isPublicKey && !!d.communityDetails)
-        readonly property var communityDetails: {
-            return root.store.getCommunityDetails(Utils.getCompressedPk(publicKey));
-        }
+        readonly property var communityDetails: root.store.getCommunityDetails(publicKey)
```

Comment:
Are you sure that there is public key not a chat key?

Reply:
Yes. `publicKey` is gotten just below. We even convert it to the right format if needed (and `getCommunityDetails` also does a conversion, that's why I removed the double conversion).